administrator/components/com_content/controllers/article.php

```diff
@@ -144,6 +144,41 @@ public function batch($model = null)
 		return parent::batch($model);
 	}
 
+	public function shareTokenGenerate($length = 16)
```

Comment:
@nikitadhiman There is no need to create your own function. Joomla has the genRandomPassword() function. See: https://github.com/roland-d/joomla-cms/blob/22fdc98d5a8eed3cf40cebb310e7402d1fb0f1f0/libraries/joomla/user/helper.php#L642-L642 You can even specify the length of the password.